scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -119,6 +139,22 @@
 parser.add_argument("files", type=str, nargs="+", help="files to plot")
 
 
+def report_find_fail(key, search_location, available, logger):
+    logger.error(f"{key} not found in {search_location}. Further processing stopped.")
+    logger.info(f"Available fields: {available}")
+    return
+
+
+def subsample(array, maxlen):
+    "Subsample array to have a maximum length of maxlen by even sampling"
+    ratio = len(array) / maxlen
+    if ratio >= 1:
+        aout = array[:: int(ratio)]
```

Comment:
I'm unfamiliar with this syntax but I checked that it works. Could the above comment be made a little more specific, something like, `"Subsample array with fixed stride to have a maximum length of maxlen"`?

Reply:
done